website/docs/reference/supported_commands.md

```diff
@@ -83,6 +83,35 @@ sidebar_position: 1
 |                   | `arrayFilters`             | ⚠️  | Unimplemented                                             |
 |                   | `hint`                     | ⚠️  | Unimplemented                                             |
 
+## Update Operators
+
+The following operators and modifiers are available in the `update` and `findAndModify` commands.
+
+| Operator          | Modifier | Status | Comments                                                  |
+|-------------------|----------|--------|-----------------------------------------------------------|
+| `$currentDate`    |          | ⚠️     | [Issue](https://github.com/FerretDB/FerretDB/issues/666)  |
+| `$inc`            |          | ⚠️     | [Issue](https://github.com/FerretDB/FerretDB/issues/666)  |
+| `$min`            |          | ⚠️     | [Issue](https://github.com/FerretDB/FerretDB/issues/666)  |
+| `$max`            |          | ⚠️     | [Issue](https://github.com/FerretDB/FerretDB/issues/666)  |
+| `$mul`            |          | ⚠️     | [Issue](https://github.com/FerretDB/FerretDB/issues/625)  |
+| `$rename`         |          | ⚠️     | [Issue](https://github.com/FerretDB/FerretDB/issues/626)  |
+| `$set`            |          | ⚠️     | [Issue](https://github.com/FerretDB/FerretDB/issues/666)  |
+| `$setOnInsert`    |          | ⚠️     | [Issue](https://github.com/FerretDB/FerretDB/issues/666)  |
```

Comment:
Hmm, I think, in general, these are implemented. But it might be possible that not all the modifiers/options are supported yet. 

In #666 we want to cover a particular case when one request can have multiple operators of the same kind (e.g. two `$set` etc).

Reply:
I mean, the ones mentioned in #666 are implemented. `$mul` and `$rename` are not.

Reply:
Yes they are implemented but are sort of blocked by #666 as multiple instances of the operator will not work as expected, hence the ⚠️. Should I remove all links to #666 then?

Reply:
Done - gave implemented operators the correct check mark as discussed.